Products/ZenModel/tests/testIpNetwork.py

```diff
@@ -254,6 +255,20 @@ def assertZPropertiesExist( network ):
         subnet24 = dmdNet.createNet("10.10.10.0", 24)
         assertZPropertiesExist(subnet24)
 
+    def testFindStaleNetOrIp(self):
+        """
+        If we have a record in cache which is not in database it 
+        should be removed from cache and return None when attempt to get it 
+        """
+        dmdNet = self.dmd.Networks
+        # Populate cache without adding object to database
+        dmdNet.get_network_cache().add_net(IpNetwork('1.1.1.0', '24'))
+        net = dmdNet.findNet('1.1.1.0', '24')
+        self.assert_(net == None)
```

Comment:
The "pythonic" way to write this test is

```
self.assertIsNone(net)
```
If, for some reason, the `assertIsNone` method is unavailable, then use the `is` operator, e.g.
```
self.assert_(net is None)
```

Reply:
done